packs/feat-effects/effect-familiar-of-keen-senses.json

```diff
@@ -0,0 +1,82 @@
+{
+    "_id": "4wJM0OA9y2gsBAh7",
+    "img": "systems/pf2e/icons/actions/Passive.webp",
+    "name": "Effect: Familiar of Keen Senses",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.familiar-abilities.onye3PN84lmca11D]{Familiar of Keen Senses}</p>\n<p>When you Cast or Sustain a hex, your familiar gains your choice of an imprecise scent, tremorsense, or wavesense, with a range of 60 feet until the start of your next turn, and it can immediately @UUID[Compendium.pf2e.actionspf2e.Item.Point Out] as a free action.</p>"
```

Comment:
The granted by link isn’t quite correct, as it wasn’t extracted to be human-readable. Looks like `.Item` is missing. Maybe Quick Insert was used to add the link?

Reply:
If using quick insert you can drag from the QI window to the editor to generate a regular link.

Reply:
> If using quick insert you can drag from the QI window to the editor to generate a regular link.

Oh, that's good to know! I'll probably start using it like that then. Just to be clear, I wasn't using it this time, but in any case the link is fixed now.